cfg/cfg.go

```diff
@@ -86,8 +91,15 @@ func init() {
 		&DisablePushingToEndpoint,
 		"disable-push",
 		false,
-		"whether pushing of data to endpoint should be disabled. "+
-			"`false` by default. Existing for testing and debugging purposes",
+		"Whether pushing of data and logs to endpoint should be disabled. "+
+			"`false` by default. Needed for testing and debugging purposes",
+	)
+	flag.BoolVar(
+		&PrintEventsOnly,
+		"print-events-only",
+		false,
+		"Output events instead of regular logs to immitate what a user can see."+
```

Comment:
I'm a bit confused here.
Docs says `Output events instead of regular logs` but actually, regular logs already contains `Output events` + `Debug logs`
```
2019/03/01 18:33:05 Downloading VS Code extension archive 'https://marketplace.visualstudio.com/_apis/public/gallery/publishers/ms-vscode/vsextensions/node-debug2/1.31.6/vspackage' for plugin 'ms-vscode.node-debug2:1.31.6' to '/tmp/vscode-extension-broker475878771/pluginArchive'
2019/03/01 18:33:05 Downloading VS Code extension for plugin 'ms-vscode.node-debug2:1.31.6'
```

Maybe it would be a bit clear if parameter would be name after log level concept? Like `logLevel=debug/info`, default `debug`. WDYT?

Reply:
It is not about log level. It shows messages representation. 
Changed to `"Output events that are usually sent Che master instead of regular logs to imitate what a user can see."+
			"`false` by default. Needed for testing and debugging purposes"`